cabal.project

```diff
@@ -32,4 +31,4 @@ source-repository-package
   location: https://github.com/CodiePP/ekg-prometheus-adapter
   tag: 1a258b6df7d9807d4c4ff3e99722223d31a2c320
 
-reorder-goals: True
+reorder-goals: False
```

Comment:
Looks unrelated. Keep in a separate PR if it's needed.

Reply:
left unchanged.